src/runtime/database.h

```diff
@@ -176,6 +184,9 @@ struct Database {
 
   std::vector<JobEdge> get_edges();
   std::vector<JobTag> get_tags();
+
+  std::vector<JobReflection> get_job_visualization();
+  std::vector<std::pair<int, long>> get_file_accesses();
```

Comment:
Are these methods const methods or do they modify the struct? they should be marked const if so

```
std::vector<JobReflection> get_job_visualization() const;
```

Reply:
Done